frontend/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Pull the Node image from Docker Hub
+FROM node:12.4.0-alpine
```

Comment:
Please update the Node.js version to **v18.15.0**
to match the project's requirements